docs/etl.md

```diff
@@ -78,6 +78,35 @@ data integration, and mission-critical applications.
 ```
 
 
+## dbt
+
+```{div}
+:style: "float: right"
+[![](https://www.getdbt.com/ui/img/logos/dbt-logo.svg){w=180px}](https://www.getdbt.com/)
+```
+
+[dbt] is an open source tool for transforming data in data warehouses using Python and
+SQL. It is an SQL-first transformation workflow platform that lets teams quickly and
+collaboratively deploy analytics code following software engineering best practices
+like modularity, portability, CI/CD, and documentation.
+
+> dbt enables data analysts and engineers to transform their data using the same
+> practices that software engineers use to build applications.
```

Comment:
![image](https://github.com/crate/crate-clients-tools/assets/453543/1be6dcce-7cad-467b-ba9d-3a053a510c1c)

We are already tracking this font size glitch. It will be fixed.
- https://github.com/crate/crate-docs-theme/issues/382